app/mixins/apollo-component-query-manager.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-apollo-client/mixins/component-query-manager';
```

Comment:
Is it a best practice to avoid putting mixins in `app/` when users can just import them from `ember-apollo-client/mixins/{name}`?